service/src/test/java/bio/terra/externalcreds/services/OAuth2ServiceTest.java

```diff
@@ -0,0 +1,76 @@
+package bio.terra.externalcreds.services;
+
+import bio.terra.common.logging.LoggingInitializer;
+import bio.terra.externalcreds.config.ProviderConfig;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.security.oauth2.client.registration.ClientRegistration;
+import org.springframework.security.oauth2.core.endpoint.OAuth2AccessTokenResponse;
+import org.springframework.security.oauth2.core.user.OAuth2User;
+import org.springframework.stereotype.Component;
+
+@SpringBootApplication(scanBasePackages = {"bio.terra.externalcreds", "bio.terra.common.logging"})
```

Comment:
We don't need the google formatted logs
```suggestion
@SpringBootApplication(scanBasePackages = "bio.terra.externalcreds")
```